devtools/cli/src/main/java/io/quarkus/cli/Build.java

```diff
@@ -33,14 +33,23 @@ public Integer call() {
             output.throwIfUnmatchedArguments(spec.commandLine());
 
             BuildSystemRunner runner = getRunner();
+            if (buildOptions.generateReport) {
+                params.add("-Dquarkus.debug.dump-build-metrics=true");
```

Comment:
`quarkus.debug.dump-build-metrics`  should be added to the config so that it doesn't trigger the unknown property warning.